test/test_nn.py

```diff
@@ -12993,7 +13024,7 @@ def __init__(self) -> None:
         self.assertEqual(list(state_dict._metadata.keys()), list(ddp_state_dict._metadata.keys()))
 
 
-instantiate_device_type_tests(TestNNDeviceType, globals(), allow_mps=True)
+instantiate_device_type_tests(TestNNDeviceType, globals())
```

Comment:
why did you remove `allow_mps` here? Please do not do this

It seems like your linter also proposed a bunch of changes, could you undo these or split them into a separate PR please?